config/routes.rb

```diff
@@ -66,6 +66,12 @@
   get '/isolation_segments/:guid', to: 'isolation_segments#show'
   delete '/isolation_segments/:guid', to: 'isolation_segments#destroy'
   put '/isolation_segments/:guid', to: 'isolation_segments#update'
+  post '/isolation_segments/:guid/relationships/organizations', to: 'isolation_segments#assign_allowed_organizations'
+  delete '/isolation_segments/:guid/relationships/organizations', to: 'isolation_segments#unassign_allowed_organizations'
+
+  get '/isolation_segments/:guid/relationships/organizations', to: 'isolation_segments#relationships_orgs'
+  # To build these relationships. OrgManagers/Admins set the isolation segment on Spaces
```

Comment:
I don't think this comment is needed. This type of information should be in specs/ API docs instead.